.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -56,11 +55,7 @@ body:
       label: Mobile Platform Information
       description: |
         Would you mind to tell us the system information about your mobile platform?
-      value: |
-        - Device: [e.g. iPhone6]
-        - OS: [e.g. iOS8.1]
-        - Browser [e.g. stock browser, safari]
-        - Version [e.g. 22]
+      placeholder: "example: Android App v0.5.9"
```

Comment:
The same change should be applied as above item.

Reply:
```suggestion
      placeholder: |
        Device, OS version, Browser or App, Logseq App version
        example: iPhone6, iOS8.1, App, v0.5.9
```